File: iOS/Component/Mix/TestPods_OC/TestPods_OC/ViewController.m
//
//  ViewController.m
//  TestPods_OC
//
//  Created by 启业云03 on 2021/3/23.
//

#import "ViewController.h"

//#import <OnlyOCDemo/AllHeader.h>
#import <OnlyOCDemo/FunctionB.h>

#import <OnlySwiftDemo/OnlySwiftDemo-Swift.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.

    self.view.backgroundColor = UIColor.orangeColor;
}

- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event {
    // 纯OC组件
//    [FunctionA printSomethingA:@"A打印"];
    [FunctionB printSomethingB:@"B打印"];
    
    // 纯Swift组件
    SwiftLibC *C = [[SwiftLibC alloc] init];
    [C show];
}


@end

File: iOS/Component/Mix/TestPods_Swift/TestPods_Swift/ViewController.swift
//
//  ViewController.swift
//  TestPods_Swift
//
//  Created by 启业云03 on 2021/3/23.
//

import UIKit
import OnlyOCDemo
import OnlySwiftDemo

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        view.backgroundColor = .orange
    }
    
    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
        // 纯OC组件
        FunctionA.printSomethingA("你大爷！")
        FunctionB.printSomethingB("我二爷！")
        
        // 纯Swift组件
        OnlySwiftDemo.SwiftLibC().show()
    }

}

